DOCUMENTATION.md

```diff
@@ -93,6 +93,14 @@ same defaults as the main SDK factory function.
 import { fromSharedOptions } from 'balena-sdk';
 const sdk = fromSharedOptions();
 ```
+
+## Trying balenaSDK in the browser
```

Comment:
It feels weird that this appears in between the documentation for the exported methods & the methods for the balena instance.
How about just using the pre-existing DOCUMENTATION.hbs and just add this section right above `{{>main}}` for simplicity? The current content can also be changed to better match the python sdk docs page
See: https://www.balena.io/docs/reference/sdk/python-sdk/

IE copy over the `## Installation` & the unpkg.com sections from the README.

Reply:
We thought about that and it looked weird to me on first thought. Check out our same conversation here: https://github.com/balena-io/docs/issues/2205

 IMHO ^^ this was waaay simpler for me to do then go hacking around the partials.